src/sparseml/pytorch/sparsification/quantization/modifier_quantization.py

```diff
@@ -476,15 +533,15 @@ def _check_quantization_update(
 
         if self._freeze_bn_stats_update_ready(epoch):
             for _, quant_module in self._modules_to_quantize:
-                quant_module.apply(torch_intrinsic.qat.freeze_bn_stats)
+                quant_module.apply(freeze_bn_stats)
             self._bn_stats_frozen = True
 
     def _enable_module_qat(self, module: Module):
         # fuse module Conv-BNs
-        if (
-            self._model_fuse_fn_name is not None
-            and self._model_fuse_fn_name != "no_fuse"
-        ):  # module class fn
+        if self.model_fuse_fn_name == "conv_bn_relus":
```

Comment:
looks like this changes the default behavior so "conv_bn_relus" needs to be passed in? to maintain recipe backwards compatibility, we need to maintain the previous behavior of None using the default function

Reply:
The default behavior still is "conv_bn_relus". self.model_fuse_fn_name is a wrapper for self._model_fuse_fn_name. I handle the None case inside the wrapper now (lines 237-254). This is needed because the tensorrt flag changes the default to 'no_fuse'.